{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/postgresclientorm\/tableobjecterror\/errordescription"]}],"sections":[],"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription"},"metadata":{"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"roleHeading":"Instance Property","modules":[{"name":"PostgresClientORM"}],"title":"errorDescription","role":"symbol","symbolKind":"property","externalID":"s:17PostgresClientORM16TableObjectErrorO16errorDescriptionSSSgvp"},"abstract":[{"text":"Inherited from ","type":"text"},{"code":"LocalizedError.errorDescription","type":"codeVoice"},{"text":".","type":"text"}],"primaryContentSections":[{"declarations":[{"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"title":"TableObjectError","role":"symbol","abstract":[],"kind":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","url":"\/documentation\/postgresclientorm\/tableobjecterror","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"TableObjectError"}],"navigatorTitle":[{"kind":"identifier","text":"TableObjectError"}],"type":"topic"},"doc://PostgresClientORM/documentation/PostgresClientORM":{"type":"topic","kind":"symbol","title":"PostgresClientORM","url":"\/documentation\/postgresclientorm","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","role":"collection","abstract":[]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/errorDescription":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","type":"topic","title":"errorDescription","kind":"symbol","role":"symbol","abstract":[],"url":"\/documentation\/postgresclientorm\/tableobjecterror\/errordescription"}}}